tests/webtk_test.py

```diff
@@ -0,0 +1,23 @@
+import urllib.request
+try:
+    import tkinter as tk
+except ImportError:
+    import Tkinter as tk
+import PIL.Image
+import PIL.ImageTk
+from tkinter import Label
+import io
+
+root = tk.Tk()
+
+
+raw_data = urllib.request.urlopen(r"https://maps.googleapis.com/maps/api/staticmap?size=600x400&path=enc%3Ayws{HvcqAAc@?EKs@ORkA|BfBtCL^FX_@f@q@|@eCzCk@h@i@^aAj@a@TCDKn@Gf@]pDWbCq@lG_@tCi@dDo@lD_AfFiC|OqA|H_BpKqDfWcAvIK~@g@fGeAbMi@nIyAnRQbCQjDGvACdG?x@H`ELv@RbBBDDLBZA^ERMTc@lBSt@KjBUlEItBMtOC|F@rDFrC\xF^zE`@|EF`@PdEDjCEbHEjBIbCGZ[zGq@dOMpDAdC?`CPhJ`@|AB@FJF\C`@OTIDKAIEEEw@rA}@nA]Vg@Fu@DaBN&key=YOUR_API_KEY").read()
```

Comment:
you can test this link on the browser or here, just change the api key after : '&key='